src/Service.Host/client/src/components/consignments/ItemsTab.js

```diff
@@ -87,6 +101,206 @@ function ItemsTab({
+
+            selectedPallet.height = height > selectedPallet.height ? height : selectedPallet.height;
+            selectedPallet.weight = weight;
+            selectedPallet.width = selectedPallet.width < 120 ? 120 : selectedPallet.width;
+            selectedPallet.depth = selectedPallet.depth < 100 ? 100 : selectedPallet.depth;
+
+            setPalletRowToBeSaved(palletNumber, true);
+        }
+    };
+
+    const okToPalletise = item => {
+        if (item && !item.palletNumber) {
+            if (item.itemType === 'I' && item.containerNumber) {
+                return false;
+            }
+
+            return true;
+        }
+
+        return false;
+    };
+
+    const addItemToPallet = (palletNumber, itemNumber) => {
+        const selectedPallet = palletData.find(a => a.palletNumber === palletNumber);
+        const currentItem = itemsData.find(a => a.itemNumber === itemNumber);
+
+        if (selectedPallet && okToPalletise(currentItem)) {
+            currentItem.palletNumber = palletNumber;
+            setItemRowToBeSaved(currentItem.itemNumber, true);
+        }
+    };
+
+    const addItemsToPallet = (palletNumber, first, last) => {
+        const selectedPallet = palletData.find(a => a.palletNumber === palletNumber);
+        if (selectedPallet) {
+            for (let i = first; i <= last; i += 1) {
+                addItemToPallet(palletNumber, i);
+            }
+
+            recalculatePallet(palletNumber);
+            setEditStatus('edit');
+            setPalletRowToBeSaved(palletNumber, true);
+            setAddToPalletNumber(0);
+        }
+    };
+
+    const addCartonToPallet = (palletNumber, containerNumber) => {
+        const selectedPallet = palletData.find(a => a.palletNumber === palletNumber);
+        const currentItem = itemsData.find(a => a.containerNumber === containerNumber);
+
+        if (selectedPallet && okToPalletise(currentItem)) {
+            currentItem.palletNumber = palletNumber;
+            setItemRowToBeSaved(currentItem.itemNumber, true);
+        }
+    };
+
+    const addCartonsToPallet = (palletNumber, first, last) => {
+        const selectedPallet = palletData.find(a => a.palletNumber === palletNumber);
+        if (selectedPallet) {
+            for (let i = first; i <= last; i += 1) {
+                addCartonToPallet(palletNumber, i);
+            }
+
+            recalculatePallet(palletNumber);
+            setEditStatus('edit');
+            setPalletRowToBeSaved(palletNumber, true);
+            setAddToPalletNumber(0);
+        }
+    };
+
+    const recalculateCarton = containerNumber => {
+        let weight = 0;
+        const setValues = item => {
+            if (item.itemType === 'C'){
+                weight += item.itemBaseWeight;
+            } else {
+                weight += item.weight;
+            }
+        };
+
+        // eslint-disable-next-line prettier/prettier
```

Comment:
what doesn't it like? I've never actually used the disable-next-line thing before